backend/api/guest_import/views.py

```
@@ -0,0 +1,231 @@
+        ).select_related("user_event")
+        if not guest_events.exists() and not guest_submissions.exists():
+            logger.warning(
+                "No guest data found for import attempt for user: %s",
+                account_user.email,
+            )
+            return no_data_found()
+
+        # Check if all the guest user's submissions are accounted for in the choices
+        if set(availability_choices.keys()) != set(
+            str(submission.user_event.public_id) for submission in guest_submissions
+        ):
```

Ruff (0.15.21)

[warning] Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)